src/storage/src/source.rs

```diff
@@ -40,3 +40,70 @@ pub use kafka::KafkaSourceReader;
 pub use source_reader_pipeline::{
     create_raw_source, RawSourceCreationConfig, SourceExportCreationConfig,
 };
+use timely::container::{PushInto, SizableContainer};
+use timely::dataflow::channels::pact::Pipeline;
+use timely::dataflow::operators::generic::builder_rc::OperatorBuilder;
+use timely::dataflow::{Scope, StreamCore};
+use timely::Container;
+
+/// Partition a stream of records into multiple streams.
+pub trait PartitionCore<G: Scope, C: Container> {
```

Comment:
This is a helper trait until https://github.com/TimelyDataflow/timely-dataflow/pull/610 is merged